Dockerfile.prod

```diff
@@ -18,7 +18,7 @@ COPY . /app
 WORKDIR /app
 
 # Installing packages
-RUN rm .yarnrc
+RUN yarn add @atlassian/sqs-queue-dlq-service
```

Comment:
The only drawback is we are not `frozen` this dep to a specific version, but I guess that's okay, since it is only used as our queue replay api, won't affect customer (in a disaster way)